x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/CompositeAggCursor.java

```diff
@@ -162,71 +163,64 @@ static void handle(
         BiFunction<SearchSourceBuilder, CompositeAggRowSet, CompositeAggCursor> makeCursor,
         Runnable retry,
         ActionListener<Page> listener,
-        Schema schema
+        boolean mightProducePartialPages
```

Comment:
Nit: `couldProducePartialPage` would be more suggestive to me (for both arg and method).